.jenkins

```diff
@@ -28,19 +28,18 @@ pipeline {
                         dockerfile {
                             filename 'Dockerfile.sycl'
                             dir 'scripts/docker'
-                            additionalBuildArgs '--build-arg BASE=intel/oneapi-basekit:devel-ubuntu18.04'
-                            label 'docker'
+                            label 'nvidia-docker && volta'
                             args '-v /tmp/ccache.kokkos:/tmp/ccache'
                         }
                     }
                     steps {
                         sh 'ccache --zero-stats'
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               cmake \
-                                -DCMAKE_BUILD_TYPE=Debug \
+                                -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                 -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="-Werror" \
+                                -DCMAKE_CXX_FLAGS="-Werror -Wno-unknown-cuda-version -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice" \
```

Comment:
We are not using all the `Kokkos` flags for building `gtest` and `printing-tool`:
```
cd /tmp/kokkos/build_sycl_cuda/algorithms/unit_tests && /opt/sycl/bin/clang++ -DGTEST_HAS_PTHREAD=0 -DGTEST_HAS_TR1_TUPLE=0 -I/tmp/kokkos/build_sycl_cuda/algorithms/unit_tests -I/tmp/kokkos/algorithms/unit_tests -I/tmp/kokkos/algorithms/unit_tests/../src -I/tmp/kokkos/tpls/gtest -Werror -Wno-unknown-cuda-version -fsycl-targets=nvptx64-nvidia-cuda-sycldevice -ggdb -O3 -DNDEBUG -o CMakeFiles/kokkosalgorithms_gtest.dir/__/__/tpls/gtest/gtest/gtest-all.cc.o -c /tmp/kokkos/tpls/gtest/gtest/gtest-all.cc
```
but we need to use `-fsycl` when `-fsycl-targets` is specified. Since building for `CUDA` is an unusual case, it didn't seem worth to hack the `-fsycl-targets` required for `CUDA` into the `Kokkos` flags but rather handle it this way.

Reply:
Is it something we could handle if we set the `Kokkos_ARCH_VOLTA70`?

Reply:
Ping

Reply:
We can certainly do that. The question for me is we actually want to support it (and if that should be part of this pull request).